### PR TITLE
Avoid access to protected class member.

### PR DESCRIPTION
**Pull request by twaugh**
Hopefully this increases the project's health according to Landscape.

### CODE REVIEW

File: osbs/http.py [hunk 1]
```diff
@@ -75,6 +75,14 @@ def __init__(self, status_code=0, content=b'', curl=None, raw_headers=None):
         self._headers = None
 
     @property
+    def raw_headers(self):
+        raise RuntimeError("Only settable")
```
Comment (TomasTomecek) [inline osbs/http.py]: Why not make them readable?

**Comment by TomasTomecek** [comment]
plus in `headers` I would read via `raw_headers` and not `_raw_headers`. Otherwise it looks great! Maybe we could exclude those complexity checkers and rump up the health to 100 :)

**Comment by twaugh** [comment]
Well, in that case, why not just change `_raw_headers` to `raw_headers`?

**Comment by TomasTomecek** [comment]
That's also possible. Doing it with `property` is the official pythonic way. I don't have no strong opinions for one way or the other. After all, these are inner guts of osbs client.

**Comment by TomasTomecek** [comment]
could you please change the access of `raw_headers` here? https://github.com/DBuildService/osbs/pull/47/files#diff-a49481931f2791e303e626d1083bc601R89 we can merge then from my pov

**Comment by twaugh** [comment]
Not sure where you mean -- the accesses from `Response.__init__()` and `Response.headers()` maybe?

**Comment by TomasTomecek** [comment]
Ah, I though I'm including line number too. Silly me!
https://github.com/DBuildService/osbs/pull/47/files#diff-a49481931f2791e303e626d1083bc601L81
```diff
- m = httplib.HTTPMessage(self._raw_headers, False)
+ m = httplib.HTTPMessage(self.raw_headers, False)
```

**Comment by twaugh** [comment]
OK. How about the access from `__init__()`?

```diff
-        self._raw_headers = raw_headers
+        self.raw_headers = raw_headers
```

**Comment by TomasTomecek** [comment]
Good catch! But I think that some static analyzer will complain that `self._raw_headers` is not defined in constructor.

**Comment by twaugh** [comment]
OK, I committed just the change you mentioned.

**Comment by TomasTomecek** [comment]
Thanks for doing this Tim!

**Pull request merged**